src/XMakeBuildEngine/Resources/Constants.cs

```diff
@@ -210,7 +210,7 @@ internal static long AssemblyTimestamp
     internal static class AvailableStaticMethods
     {
         /// <summary>
-        /// Static methods that are allowed in constants. Key = Type or Type::Method, Value = AssemblyQualifiedTypeName (where null = mscorlib)
+        /// Static methods that are allowed in constants. Key = Type or Type::Method, Value = Tuple of AssemblyQualifiedTypeName (where null = mscorlib) or the actual type object
```

Comment:
It's a union type where at least one has to be non-null. If both are null, the code fallbacks to Type.GetType(`the type string that the user put in the build script`)